tests/FakeItEasy.Specs/FluentApiSpecs.cs

```diff
@@ -0,0 +1,41 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+
+    using FluentAssertions;
+    using Xbehave;
+    using Xunit;
+
+    /// <summary>
+    /// Check certain fluent API syntax invariants.
+    /// </summary>
+    public static class FluentApiSpecs
+    {
+        [Scenario]
+        public static void AAndAnShouldHaveTheSameMembers(IEnumerable<string> aMembers, IEnumerable<string> anMembers)

```

Comment:
Good idea!

Reply:
Oh. I half forgot about this! I ended up not changing `A<T>` and `An<T>`, so this doesn't provide a tonne of value in this PR. No harm though. I"ll drop if you want.

Reply:
No, keep it!